Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG GOVERS=1.8.3
+FROM golang:$GOVERS
```

Comment:
why do you need to make Go version configurable like that? (Using ARG in FROM breaks lots of stuff eg content trust)

Reply:
ah wasn't aware it would break content trust.
thought it'd be handy so the go version would only need updating in one place (ie. Makefile).
happy to remove it.